### ISSUE TITLE
Maybe jquery 3?

### ISSUE THREAD

**Issue by bilazik** [almasaeed2010/AdminLTE#1105]
Hi. This template is amazing. It's really  gooooood.
What do you think about updating to jQuery 3 version and newest bootstrap?
It may be cool. 
I can\want to help with it, because I like AdminLTE, but I cant use it on production, cause need jQuery 3 support.

**Comment by karmrajzala** [comment]
@bilazik , Yes you have to wait until new version will not release. Or you can use latest Jquery version though assestManager configuration like this :

`'assetManager' => [
    'bundles' => [
        'yii\web\JqueryAsset' => [
            'js'=>[YOUR-JS-PATH]
        ],
        'yii\bootstrap\BootstrapPluginAsset' => [
            'js'=>[YOUR-JS-PATH]
        ],
        'yii\bootstrap\BootstrapAsset' => [
            'css' => [YOUR-CSS-PATH],
        ],

    ],
],`
